### PR TITLE
Update README: hidraw: minimum supported kernel

### PR DESCRIPTION
**Pull request by Youw**

### CODE REVIEW

**Comment by Youw** [comment]
In Addition to https://github.com/libusb/hidapi/pull/39
triggered by this discussion: https://github.com/libusb/hidapi/commit/6f79625abbddec29cace76b2d1409a206db4491f#commitcomment-34370237

**Comment by z3ntu** [comment]
`linux/README.txt` can probably be mostly dropped (and the rest reworked) as the document was last updated in 2010 (excluding typo fixes, etc) and covers kernels < 2.6.39.

**Comment by Youw** [comment]
updated

**Comment by z3ntu** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -43,13 +43,12 @@ tradeoffs, and the functionality supported is slightly different.
...
 
-This back-end uses the hidraw interface in the Linux kernel.  While this
-back-end will support both USB and Bluetooth, it has some limitations on
-kernels prior to 2.6.39, including the inability to send or receive feature
-reports.  In addition, it will only communicate with devices which have
-hidraw nodes associated with them.  Keyboards, mice, and some other devices
-which are blacklisted from having hidraw nodes will not work. Fortunately,
-for nearly all the uses of hidraw, this is not a problem.
+This back-end uses the hidraw interface in the Linux kernel. It requires 
+kernel version at least 2.6.39, to build.  In addition, it will only
```
Comment (Qbicz) [inline README.md]:
```suggestion
kernel version at least 2.6.39 to build. In addition, it will only 
```

File: README.md [hunk 2]
```diff
@@ -43,13 +43,12 @@ tradeoffs, and the functionality supported is slightly different.
...
 __Linux/hidraw__ (`linux/hid.c`):
 
-This back-end uses the hidraw interface in the Linux kernel.  While this
-back-end will support both USB and Bluetooth, it has some limitations on
-kernels prior to 2.6.39, including the inability to send or receive feature
-reports.  In addition, it will only communicate with devices which have
-hidraw nodes associated with them.  Keyboards, mice, and some other devices
-which are blacklisted from having hidraw nodes will not work. Fortunately,
-for nearly all the uses of hidraw, this is not a problem.
+This back-end uses the hidraw interface in the Linux kernel. It requires
```
Comment (Qbicz) [inline README.md]: Please leave the note about USB and Bluetooth. It is important, as opposed to libusb backend which only supports USB.

**Comment by Youw** [comment]
going to ignore failed build (unrelated to this change) and merge

**Pull request merged**